src/app/dataset-view/dataset-view-menu/dataset-view-menu.component.ts

```diff
@@ -29,7 +40,16 @@ export class DatasetViewMenuComponent implements OnInit {
 
     private sideNavHelper: SideNavHelper;
 
-    constructor(private clipboard: Clipboard, private datasetPermissionsServices: DatasetPermissionsService) {}
+    constructor(
+        private clipboard: Clipboard,
+        private datasetPermissionsServices: DatasetPermissionsService,
+        private lineageGraphHeigthService: LineageGraphHeigthService,
+    ) {}
+    ngAfterViewInit(): void {
```

Comment:
Blank lines are free, also make it explicitly public

Reply:
Fixed